### ISSUE TITLE
How to encode a space intentionally left by the scribe?

### ISSUE THREAD

**Issue by ppasedach** [chchch/upama#6]
![Selection_134](https://user-images.githubusercontent.com/11517856/123853497-d1149c80-d91d-11eb-9cb2-1769817c76d1.png)
The scribe left a space between _ca_ and _hemagiri°_, probably because the folio is damaged on the lower edge, and he wanted to keep some distance to that. If one wants to record this in the transcript, what would be the best way of doing so?

**Comment by wujastyk** [comment]
I would do this:

```
<gap extent="4" unit="char" reason="indent in writing surface"/>
```

> `<gap>` (gap) indicates a point where material has been omitted in a transcription, whether for editorial reasons described in the TEI header, as part of sampling practice, or because the material is illegible, invisible, or inaudible. [[3.5.3 Additions, Deletions, and Omissions](https://tei-c.org/release/doc/tei-p5-doc/en/html/CO.html#COEDADD)]

I think this is right, although the description sounds like it's the transcriber's decision, rather than a description of the source. But "illegible, invisible" would seem to cover it

**Comment by chchch** [comment]
Hmm, I think technically that would be a <code><space/></code>, rather than <code><gap/></code>. From what I understand, in the TEI spec, a gap is where the transcriber has left something untranscribed, whereas a space describes a space that's actually in the source document.

By the way, I've finally realized that I've been using <code>@reason</code> wrong... I use it as you do, giving a descriptive reason, but I think TEI wants a whitespace-separated list of "words", e.g. <code>reason="illegible damaged"</code>. I don't think it's a big problem though; if, in the future, you want to search for all gaps that are due to the text being "illegible", it'll work whether you write <code>reason="illegible"</code> or <code>reason="the text is illegible here"</code>.

**Comment by wujastyk** [comment]
You're right, Charles.  `<space>` is the appropriate element, not `<gap>`.  My bad

> The author or scribe may have left space for a word, or for an initial capital, and for some reason the word or capital was never supplied and the space left empty. The presence of significant space in the text being transcribed may be indicated by the space element. 
-- [TEI 11.4.1](https://www.tei-c.org/release/doc/tei-p5-doc/en/html/PH.html#PHSP)

**Comment by ppasedach** [comment]
https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-space.html does not mention  `@reason` as a possible attribute. We do use the space element for <space type="binding hole"/>, so maybe something like <space type="material damage"> could work?

**Comment by wujastyk** [comment]
That's true.  We might want to add @reason to the DTD (TEI is a way of defining useful tags, not a set of hard rules).  Or use @type, adaptively, perhaps?

**Comment by chchch** [comment]
It looks like the DHARMA project suggests ``<space type="defect"/>`` here: https://halshs.archives-ouvertes.fr/halshs-02888186/document (page 54)
